admin-api/authorization_server/_index.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Authorization Server
+menuTitle: Authorization Server
+weight: 4
+---
+
+# Authorization Server
+
+The Admin API is based on the OAuth2 protocol, PrestaShop is always the resource server in charge of providing access to the resources.
+
+However, it can interact with either an external authorization server or act as the authorization server itself (you can even combine both features if you need to).
+
+### Multiple authorization servers
+
+To handle multiple authorization servers we provide an interface `PrestaShop\PrestaShop\Core\Security\OAuth2\AuthorisationServerInterface`:
+
+```php
+namespace PrestaShop\PrestaShop\Core\Security\OAuth2;
+
+use Symfony\Component\HttpFoundation\Request;
+
+/**
+ * To integrate an authorization server you must implement this interface, the methods are bridges than call the authorization server
+ * to ensure the access token is valid, if so it can return a representation of the user using the JwtTokenUser DTO.
+ */
+interface AuthorisationServerInterface
+{
+    /**
+     * For each request received the resource server loops through all the available servers implementing this interface
+     * and uses this method to detect which one matches with the provided access token. Your implementation of each authorization
+     * server must be able to recognize an access token it created, usually relying on the issuer saved in the metadata included
+     * in the JWT token (no convention is forced, each authorization server may store this info differently as long as it can
+     * recognize itself).
```

Comment:
```suggestion
     * For each request received, the resource server loops through all the available servers implementing this interface
     * and uses this method to detect which one matches with the provided access token. Your implementation of each authorization
     * server must be able to recognize an access token it created, usually relying on the issuer saved in the metadata included
     * in the JWT token (no convention is forced, each authorization server may store this info differently as long as it can
     * recognize itself).
```